### PR TITLE
Add new event type E_MOVE to typelibrary with detailed information.

### PR DESCRIPTION
**Pull request by franz-hoepfinger-4diac**

### CODE REVIEW

File: data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN" Type="ANY" Comment="input"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="OUT" Type="ANY" Comment="input = output"/>
+		</OutputVars>
+	</InterfaceList>
+	<Attribute Name="GenericClassName" Value="'GEN_FORTE_E_MOVE'"/>
```
Comment (azoitl) [inline data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt]: This attribute should only be set if the type is on FORTE a generic one with a flexible number of in and outputs. As this is not the case please remove it.

**Review by azoitl — CHANGES_REQUESTED** [review]

File: data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt (hunk 1 above)
Comment (diplfranzhoepfinger) [reply data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt]: Ah. Ok. But for F_MOVE this is the case?

**Comment by franz-hoepfinger-4diac** [comment]
@azoitl: Changed.

File: data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt (hunk 1 above)
Comment (franz-hoepfinger-4diac) [reply data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt]: https://github.com/eclipse-4diac/4diac-ide/pull/688

Comment (azoitl) [reply data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt]: Yes F_MOVE needs it!!!

**Review by azoitl — APPROVED** [review]

File: data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt (hunk 1 above)
Comment (diplfranzhoepfinger) [reply data/typelibrary/events-1.0.0/typelib/E_MOVE.fbt]: Interesting to know.

**Pull request merged**